decidim-core/db/seeds.rb

```diff
@@ -31,12 +31,12 @@
       from: "#{smtp_label} <#{smtp_email}>",
       from_email: smtp_email,
       from_label: smtp_label,
-      user_name: ENV["SMTP_USERNAME"] || Faker::Twitter.unique.screen_name,
-      encrypted_password: Decidim::AttributeEncryptor.encrypt(ENV["SMTP_PASSWORD"] || Faker::Internet.password(min_length: 8)),
-      address: ENV["SMTP_ADDRESS"] || ENV["DECIDIM_HOST"] || "localhost",
-      port: ENV["SMTP_PORT"] || ENV["DECIDIM_SMTP_PORT"] || "25"
+      user_name: ENV.fetch("SMTP_USERNAME", nil) || Faker::Twitter.unique.screen_name,
+      encrypted_password: Decidim::AttributeEncryptor.encrypt(ENV.fetch("SMTP_PASSWORD", nil) || Faker::Internet.password(min_length: 8)),
+      address: ENV.fetch("SMTP_ADDRESS", nil) || ENV.fetch("DECIDIM_HOST", nil) || "localhost",
```

Comment:
I think better would be:
```suggestion
      address: ENV.fetch("SMTP_ADDRESS", ENV.fetch("DECIDIM_HOST", "localhost")),
```